src/discussions/learners/test-utils.js

```diff
@@ -50,3 +52,11 @@ export async function setupPostsMockResponse({
   await executeThunk(fetchUserPosts(courseId, { filters }), store.dispatch, store.getState);
   return store.getState().threads;
 }
+
+export async function setUpPrivilages(axiosMock, store, hasModerationPrivileges) {
+  axiosMock.onGet(`${getCourseConfigApiUrl()}${courseId}/`).reply(200, {
+    has_moderation_privileges: hasModerationPrivileges,
+  });
+  axiosMock.onGet(`${getCourseConfigApiUrl()}${courseId}/settings`).reply(200, {});
```

Comment:
Can you please modify this URL in come constant like **getDiscussionsSettingsURL** in `frontend-app-discussions/src/discussions/data/api.js` then export it from there, because this url is used multiple times. and change where it is used in all files